v1.1/install-cockroachdb.html

```diff
@@ -374,6 +374,13 @@ <h2 id="whats-next">What's Next?</h2>
         <td>Autoconf</td>
         <td>Version 2.68 or higher is required.</td>
       </tr>
+      <tr>
+        <td>Terminfo / NCurses development libraries</td>
+        <td>This may be
+            part of a ncurses development package (e.g. `libtinfo-dev` on
```

Comment:
I think you'll need to wrap `libtinfo-dev` in `<code>libtinfo-dev</code>` (also `ncurses-devel`).

Reply:
Done.